workers/unity/Packages/com.improbable.gdk.core/Subscriptions/EntityGameObjectLinker.cs

```diff
@@ -72,10 +72,10 @@ public EntityGameObjectLinker(World world)
             linkedGameObjects.Add(gameObject);
 
             linkedComponent.IsValid = true;
-            linkedComponent.Linker = this;
             linkedComponent.EntityId = entityId;
             linkedComponent.World = world;
             linkedComponent.Worker = workerSystem;
+            linkedComponent.Linker = this;
```

Comment:
Why? Does the ordering here actually matter?

Reply:
Side effect

Reply:
The ordering does not matter